downstream/assemblies/hub/assembly-hub-overview.adoc

```diff
@@ -1,20 +1,21 @@
 ifdef::context[:parent-context: {context}]
 
 [id="hub-overview"]
-= Overview of {HubName} and {PrivateHubName}
+= Distinctions between {HubName} and {PrivateHubName}
 
 :context: hub-overview
 
 [role="_abstract"]
-Two versions of {HubName} are provided with {PlatformName}; {HubName} and {PrivateHubName}. Both versions provide a central location where automation content can be located, downloaded, and integrated into your Ansible automation.
+{PlatformName} uses an {HubName} as a central location for automation content that you can download and integrate into your Ansible automation. Two types of hubs are available:
 
-Ansible content can be broken up into three main categories:
+{HubNameStart}:: Hosted by Red Hat, it contains only Red Hat supported or certified content.
+{PrivateHubNameStart}:: This is a self-hosted content management system. You can use it to access and manage all types of Ansible content and choose which Ansible content collections and versions are made available to your automation consumers.
 
-* Community content found in {Galaxy}
-* Red Hat certified and supported content found in {HubName} on the link:https://console.redhat.com/ansible/automation-hub[Red Hat Hybrid Cloud Console]
-* Private content created and curated by an organization and shared locally
+With a {PrivateHubName}, you can access these three types of content:
 
-The distinction between {HubName} and {PrivateHubName} is that {HubName} allows you to access only Red Hat supported or certified content, and {PrivateHubName}, as a self-hosted content management system, allows you to access and manage of all three types of content, choosing which Ansible content collections and versions are made available to your automation consumers.
+* Red Hat certified and supported content found in {HubName} on the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console]
```

Comment:
The sentences in these last three bullet points should end with periods for consistency with the first two bullets (and also bc they're complete sentences).